Assets/HoloToolkit/Utilities/Scripts/ManualCameraControl.cs

```diff
@@ -28,6 +28,18 @@ public enum MouseButton
     public MouseButton MouseLookButton = MouseButton.Shift;
     public bool IsControllerLookInverted = true;
 
+    public enum ControlMode
```

Comment:
I like this. Having both enum and class inside the HoloToolkit.Unity for better discoverability.